test_runner/performance/test_bulk_update.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+from fixtures.neon_fixtures import NeonEnvBuilder, wait_for_last_flush_lsn
+
+
+#
+# Benchmark estimating effect of prefetch on bulk update operations
+#
+@pytest.mark.timeout(10000)
+@pytest.mark.parametrize("fillfactor", [10, 50, 100])
```

Comment:
@MMeent suggested to add it. I think that it may be reasonable because different fill factor can affect locality of changes performed by update (whether updated record is written at at same page or at some new page).

Reply:
> I don't think this PR should include any Postgres changes at all. This is supposed to just add a new performance test.
It is not including any postgres changes - just bumping postgres version.
Create separate PR just for adding test and one more for bumping postgres version? A lot of stupid work with few sense IMHO:(

Reply:
> > I don't think this PR should include any Postgres changes at all. This is supposed to just add a new performance test.
> It is not including any postgres changes - just bumping postgres version.
> Create separate PR just for adding test and one more for bumping postgres version? A lot of stupid work with few sense IMHO:(

I don't think this PR should bump the postgres version. Why would it? The point it just to add a new test.

Reply:
> @MMeent suggested to add it. I think that it may be reasonable because different fill factor can affect locality of changes performed by update (whether updated record is written at at same page or at some new page).

Ok, I see. I think a small fillfactor, where all the updates are HOT updates, is the worst case to demonstrate the effect of invalidated prefetch requests. But I agree it makes sense to test non-HOT updates too, we might find other surprises with that.